### ISSUE TITLE
Sonoff SNZB-01 Unable to Pair or Add to group

### ISSUE THREAD

**Issue by GeoCyberwolf** [dresden-elektronik/deconz-rest-plugin#3490]
## Describe the bug
Unable to add Sonoff SNZB-01 Wireless switch to group in 2.05.84. I have updated to the latest beta version 2.05.86 and still could not add the switch to a group. I also can't pair the switches in 2.05.86, I can see the device announce in the logs when I reset the switch but it does not pair.

## Steps to reproduce the behavior
Try to add the switch to a group, switch does not show in the list of available switches. 

Try to pair the switch, switch does not pair.

## Expected behavior
Switch is available to add to groups
Switch pairs.

## Screenshots
![image](https://user-images.githubusercontent.com/54069743/97040161-7bede300-1565-11eb-9d5e-65a1fc3e3f7b.png)
## Environment
 - Host system: Rapberry Pi 3B
 - Running method: Raspbian
 - Firmware version: 26520700
 - deCONZ version: 2.05.86
 - Device: RaspBee II
 - Is there any other USB or serial devices connected to the host system? No

## deCONZ Logs

> pi@deconz:~ $ deCONZ --dbg-info=1
> QCoreApplication::arguments: Please instantiate the QApplication object first
> libEGL warning: DRI2: failed to authenticate
> libpng warning: iCCP: known incorrect sRGB profile
> 19:10:45:739 HTTP Server listen on address 0.0.0.0, port: 80, root: /usr/share/deCONZ/webapp/
> 19:10:45:753 CTRL. 3.27.219:10:45:988 dev /dev/ttyS0 (/dev/ttyS0)
> 19:10:45:988 ZCLDB init file /home/pi/.local/share/dresden-elektronik/deCONZ/zcldb.txt
> 19:10:46:522 parent process bash
> 19:10:46:523 gw run mode: normal
> 19:10:46:523 GW sd-card image version file does not exist: /home/pi/.local/share/dresden-elektronik/deCONZ/gw-version
> 19:10:46:523 sd-card cid: 035344534333324780ffffffff014509
> 19:10:46:555 DB sqlite version 3.27.2
> 19:10:46:559 DB PRAGMA page_count: 41
> 19:10:46:559 DB PRAGMA page_size: 4096
> 19:10:46:559 DB PRAGMA freelist_count: 2
> 19:10:46:559 DB file size 167936 bytes, free pages 2
> 19:10:46:560 DB PRAGMA user_version: 7
> 19:10:46:560 DB cleanup
> 19:10:46:565 DB create temporary views
> 18:10:46:660 started websocket server at port 443
> 18:10:46:674 [INFO] - Found file containing button maps. Parsing data...
> 18:10:46:682 [ERROR] - Button map 'sunricherCCTMap' in JSON file has no assigned ModelIDs. Skip loading button map...
> 18:10:46:697 [INFO] - Button maps loaded.
> 18:10:46:702 dlg action: Read binding table
> 18:10:46:703 found node plugin: libde_rest_plugin.so - REST API Plugin
> 18:10:46:714 found node plugin: libde_signal_plugin.so - Signal Monitor Plugin
> 18:10:46:777 found node plugin: libstd_otau_plugin.so - STD OTAU Plugin
> 18:10:46:837 dev /dev/ttyS0 (/dev/ttyS0)
> 18:10:47:213 New websocket 10.13.37.22:60295 (state: 3) 
> 18:10:47:258 New websocket 10.13.37.22:60297 (state: 3) 
> 18:10:47:930 dev /dev/ttyS0 (/dev/ttyS0)
> 18:10:47:933 COM check bootloader
> 18:10:49:941 COM timout query bootloader, assume application
> 18:10:50:011 Device firmware version 0x26520700 RaspBee II
> 18:10:50:147 unlocked max nodes: 200
> 18:10:50:329 Device protocol version: 0x010B
> 18:10:50:367 new node - ext: 0x00212effff052e3b, nwk: 0x0000
> 18:10:50:573 Tuya debug 7 : Missing manufacture name for 0x00124b002228c125
> 18:10:50:573 Tuya debug 7 : Missing manufacture name, till missing in DB.
> 18:10:50:574 SensorNode 3 set node 0x00124b002228c125
> 18:10:50:577 SensorNode 2 set node 0x00124b002228c125
> 18:10:50:581 Tuya debug 7 : Missing manufacture name for 0x00124b002228c125
> 18:10:50:581 Tuya debug 7 : Missing manufacture name, till missing in DB.
> 18:10:50:589 Tuya debug 7 : Missing manufacture name for 0x00124b002228d384
> 18:10:50:589 Tuya debug 7 : Missing manufacture name, till missing in DB.
> 18:10:50:589 SensorNode 5 set node 0x00124b002228d384
> 18:10:50:592 SensorNode 4 set node 0x00124b002228d384
> 18:10:50:596 Tuya debug 7 : Missing manufacture name for 0x00124b002228d384
> 18:10:50:596 Tuya debug 7 : Missing manufacture name, till missing in DB.
> 18:10:50:601 Tuya debug 7 : Missing manufacture name for 0x00124b001f90f55a
> 18:10:50:602 Tuya debug 7 : Missing manufacture name, till missing in DB.
> 18:10:50:603 SensorNode 6 set node 0x00124b001f90f55a
> 18:10:50:607 Tuya debug 7 : Missing manufacture name for 0x00124b001f90f55a
> 18:10:50:607 Tuya debug 7 : Missing manufacture name, till missing in DB.
> 18:10:50:610 Tuya debug 7 : Missing manufacture name for 0x00124b001f90147d
> 18:10:50:610 Tuya debug 7 : Missing manufacture name, till missing in DB.
> 18:10:50:610 SensorNode 7 set node 0x00124b001f90147d
> 18:10:50:615 Tuya debug 7 : Missing manufacture name for 0x00124b001f90147d
> 18:10:50:615 Tuya debug 7 : Missing manufacture name, till missing in DB.
> 18:10:50:622 LightNode 2: Dimmer switch 2 added
> 18:10:50:623 SensorNode 9 set node 0x00158d000423ded9
> 18:10:50:648 SensorNode 8 set node 0x00158d000444f976
> 18:10:50:657 LightNode 3: Desk backlight added
> 18:10:50:662 LightNode 4: smallboye lamp added
> 18:10:50:669 LightNode 5: Hallway 1 added
> 18:10:50:677 LightNode 6: Hallway 2 added
> 18:10:50:686 LightNode 7: Hallway 3 added
> 18:10:50:692 LightNode 8: Bigboye lamp added
> 18:10:50:697 LightNode 9: WolfDen added
> 18:10:50:701 SensorNode 10 set node 0x00178801021a4950
> 18:10:50:715 LightNode 10: Bathroom added
> 18:10:50:718 SensorNode 11 set node 0x0017880104f44026
> 18:10:50:748 CTRL [00:12:4b:00:22:29:29:06] not viable in zll.db, skip load cache (101)
> 18:10:50:749 CTRL [00:12:4b:00:22:29:31:45] not viable in zll.db, skip load cache (101)
> 18:10:50:983 Current channel 11
> 18:10:51:023 CTRL ANT_CTRL 0x03
> 18:10:51:109 Device protocol version: 0x010B
> 18:10:51:239 Current channel 11
> 18:10:51:284 CTRL ANT_CTRL 0x03
> 18:10:51:410 device announce 0x00124B0022293145 (0xD177) mac capabilities 0x80
> 18:10:51:411 set fast probe address to 0x00124B0022293145 (0xD177)
> 18:10:51:411 FP indication 0x0000 / 0x0013 (0x00124B0022293145 / 0xD177)
> 18:10:51:411                       ...     (0x00124B0022293145 / 0xD177)
> 18:10:51:411 device announce 0x00124B0022293145 (0xD177) mac capabilities 0x80
> 18:10:51:450 [INFO] - No button handler for: RWL021 endpoint: 0x02 cluster: 0x0001 command: 0x0A payload[0]: 021
> 18:10:51:450 ZCL attribute report 0x0017880104F44026 for cluster: 0x0001, ep: 0x02, frame control: 0x08, mfcode: 0x0000 
> 18:10:51:475 [INFO] - No button handler for: TH01 endpoint: 0x01 cluster: 0x0405 command: 0x0A payload[0]: 000
> 18:10:51:475 ZCL attribute report 0x00124B002228D384 for cluster: 0x0405, ep: 0x01, frame control: 0x18, mfcode: 0x0000 
> 18:10:51:476 discard double entry in binding queue (size: 2) for for 0x00124B002228D384, cluster 0x0001
> 18:10:51:794 send permit join, duration: 59
> 18:10:51:794 send GP proxy commissioning mode
> 18:10:51:894 dev /dev/ttyS0 (/dev/ttyS0)
> 18:10:51:930 GW update firmware found: /usr/share/deCONZ/firmware/deCONZ_ConBeeII_0x26660700.bin.GCF
> 18:10:51:930 GW firmware version: 0x26520700
> 18:10:52:162 Announced to internet https://phoscon.de/discover
> 18:10:52:162 discovery server date: Fri, 23 Oct 2020 18:10:52 GMT
> 18:10:52:162 	 local time seems to be ok
> 18:10:52:162 discovery found version 2.04.35 for update channel stable
> 18:10:52:292 [1] get node descriptor for 0x00124b0022293145
> 18:10:54:930 Search sensors done
> 18:10:55:792 send permit join, duration: 0
> 18:10:55:793 send GP proxy commissioning mode
> 18:10:58:264 New websocket 10.13.37.200:36348 (state: 3) 
> 18:10:59:337 New websocket 10.13.37.97:65238 (state: 3) 
> 18:10:59:379 New websocket 10.13.37.97:65239 (state: 3) 
> 18:11:01:848 dev /dev/ttyS0 (/dev/ttyS0)
> 18:11:01:848 GW firmware version: 0x26520700
> 18:11:04:280 LightNode 1: Configuration tool 1 added
> 18:11:11:847 dev /dev/ttyS0 (/dev/ttyS0)
> 18:11:11:848 GW firmware version: 0x26520700
> 18:11:21:889 dev /dev/ttyS0 (/dev/ttyS0)
> 18:11:21:892 GW firmware version: 0x26520700
> 18:11:31:047 [INFO] - No button handler for: TH01 endpoint: 0x01 cluster: 0x0405 command: 0x0A payload[0]: 000
> 18:11:31:047 ZCL attribute report 0x00124B002228C125 for cluster: 0x0405, ep: 0x01, frame control: 0x18, mfcode: 0x0000 
> 18:11:31:876 dev /dev/ttyS0 (/dev/ttyS0)
> 18:11:31:877 GW firmware version: 0x26520700
> 18:11:41:852 dev /dev/ttyS0 (/dev/ttyS0)
> 18:11:41:853 GW firmware version: 0x26520700
> 18:11:45:720 Websocket disconnected 10.13.37.22:60295, state: 0, close-code: 1001, reason: 
> 18:11:45:840 Current channel 11
> 18:11:45:861 Device TTL 5592 s flags: 0x7
> 19:11:46:178 send permit join, duration: 0
> 19:11:46:178 send GP proxy commissioning mode
> 19:11:46:931 New websocket 10.13.37.22:60314 (state: 3) 
> 19:11:47:921 Websocket disconnected 10.13.37.22:60314, state: 0, close-code: 1001, reason: 
> 19:11:48:500 ZCL attribute report 0x680AE2FFFEE3CA83 for cluster: 0x0006, ep: 0x01, frame control: 0x08, mfcode: 0x0000 
> 19:11:48:521 ZCL attribute report 0x680AE2FFFEE3CA83 for cluster: 0x0008, ep: 0x01, frame control: 0x08, mfcode: 0x0000 
> 19:11:48:545 ZCL attribute report 0x680AE2FFFEE3CA83 for cluster: 0x0300, ep: 0x01, frame control: 0x08, mfcode: 0x0000 
> 19:11:49:800 New websocket 10.13.37.22:60315 (state: 3) 
> 19:11:50:280 dev /dev/ttyS0 (/dev/ttyS0)
> 19:11:50:281 GW firmware version: 0x26520700
> 19:11:55:177 send permit join, duration: 59
> 19:11:55:178 send GP proxy commissioning mode
> 19:11:55:913 ZCL attribute report 0x680AE2FFFEE34F1E for cluster: 0x0300, ep: 0x01, frame control: 0x08, mfcode: 0x0000 
> 19:11:58:822 Search sensors done
> 19:11:59:178 send permit join, duration: 0
> 19:11:59:178 send GP proxy commissioning mode
> 19:12:00:178 send permit join, duration: 59
> 19:12:00:178 send GP proxy commissioning mode
> 19:12:00:305 dev /dev/ttyS0 (/dev/ttyS0)
> 19:12:00:306 GW firmware version: 0x26520700
> 19:12:02:177 send permit join, duration: 0
> 19:12:02:178 send GP proxy commissioning mode
> 19:12:02:445 Search sensors done
> 19:12:04:201 send permit join, duration: 0
> 19:12:04:201 send GP proxy commissioning mode
> 19:12:04:208 Search sensors done
> 19:12:05:528 ZCL attribute report 0x680AE2FFFEE34F1E for cluster: 0x0300, ep: 0x01, frame control: 0x08, mfcode: 0x0000 
> 19:12:06:178 send permit join, duration: 59
> 19:12:06:178 send GP proxy commissioning mode
> 19:12:07:157 Search sensors done
> 19:12:07:178 send permit join, duration: 0
> 19:12:07:179 send GP proxy commissioning mode
> 19:12:08:165 0xD177 nwk changed to 0x4A2D
> 19:12:09:198 saved node state in 4 ms
> 19:12:09:214 sync() in 16 ms
> 19:12:09:215 send permit join, duration: 0
> 19:12:09:216 send GP proxy commissioning mode
> 19:12:09:216 Search sensors done
> 19:12:10:178 send permit join, duration: 59
> 19:12:10:178 send GP proxy commissioning mode
> 19:12:10:247 dev /dev/ttyS0 (/dev/ttyS0)
> 19:12:10:248 CTRL skip polling while permit join is set
> 19:12:10:263 GW firmware version: 0x26520700
> 19:12:10:717 CTRL skip polling while permit join is set
> 19:12:11:723 CTRL skip polling while permit join is set
> 19:12:12:191 send permit join, duration: 0
> 19:12:12:191 send GP proxy commissioning mode
> 19:12:12:739 Search sensors done
> 19:12:14:178 send permit join, duration: 59
> 19:12:14:179 send GP proxy commissioning mode
> 19:12:14:742 CTRL skip polling while permit join is set
> 19:12:15:745 CTRL skip polling while permit join is set
> 19:12:16:750 CTRL skip polling while permit join is set
> 19:12:17:178 send permit join, duration: 0
> 19:12:17:178 send GP proxy commissioning mode
> 19:12:17:480 Search sensors done
> 19:12:19:177 send permit join, duration: 0
> 19:12:19:178 send GP proxy commissioning mode
> 19:12:19:182 Search sensors done
> 19:12:19:259 ZCL attribute report 0x680AE2FFFEE34C9B for cluster: 0x0006, ep: 0x01, frame control: 0x08, mfcode: 0x0000 
> 19:12:19:329 ZCL attribute report 0x680AE2FFFEE34C9B for cluster: 0x0008, ep: 0x01, frame control: 0x08, mfcode: 0x0000 
> 19:12:19:368 ZCL attribute report 0x680AE2FFFEE34C9B for cluster: 0x0300, ep: 0x01, frame control: 0x08, mfcode: 0x0000 
> 19:12:20:178 send permit join, duration: 59
> 19:12:20:178 send GP proxy commissioning mode
> 19:12:20:270 dev /dev/ttyS0 (/dev/ttyS0)
> 19:12:20:271 GW firmware version: 0x26520700
> 19:12:20:770 CTRL skip polling while permit join is set
> 19:12:21:723 Search sensors done
> 19:12:21:781 CTRL skip polling while permit join is set
> 19:12:22:179 send permit join, duration: 0
> 19:12:22:179 send GP proxy commissioning mode
> 19:12:23:178 send permit join, duration: 59
> 19:12:23:178 send GP proxy commissioning mode
> 19:12:24:177 send permit join, duration: 0
> 19:12:24:177 send GP proxy commissioning mode
> 19:12:24:436 Search sensors done
> 19:12:25:177 send permit join, duration: 59
> 19:12:25:177 send GP proxy commissioning mode
> 19:12:26:368 updated fingerprint for sensor Geo Environment
> 19:12:26:369 updated fingerprint for sensor Geo Environment
> 19:12:26:369 [INFO] - No button handler for: TH01 endpoint: 0x01 cluster: 0x0402 command: 0x0A payload[0]: 000
> 19:12:26:369 ZCL attribute report 0x00124B002228D384 for cluster: 0x0402, ep: 0x01, frame control: 0x18, mfcode: 0x0000 
> 19:12:27:212 send permit join, duration: 0
> 19:12:27:213 send GP proxy commissioning mode
> 19:12:27:976 Search sensors done
> 19:12:29:178 send permit join, duration: 59
> 19:12:29:178 send GP proxy commissioning mode
> 19:12:30:137 Search sensors done
> 19:12:30:178 send permit join, duration: 0
> 19:12:30:178 send GP proxy commissioning mode
> 19:12:30:279 dev /dev/ttyS0 (/dev/ttyS0)
> 19:12:30:282 GW firmware version: 0x26520700
> 19:12:31:203 send permit join, duration: 59
> 19:12:31:204 send GP proxy commissioning mode
> 19:12:32:177 send permit join, duration: 0
> 19:12:32:178 send GP proxy commissioning mode
> 19:12:32:539 0x4A2D nwk changed to 0x4AD3
> 19:12:32:540 device announce 0x00124B0022293145 (0x4AD3) mac capabilities 0x80
> 19:12:32:540 set fast probe address to 0x00124B0022293145 (0x4AD3)
> 19:12:32:540 FP indication 0x0000 / 0x0013 (0x00124B0022293145 / 0x4AD3)
> 19:12:32:540                       ...     (0x00124B0022293145 / 0x4AD3)
> 19:12:32:540 device announce 0x00124B0022293145 (0x4AD3) mac capabilities 0x80
> 19:12:32:729 Search sensors done
> 19:12:34:177 send permit join, duration: 59
> 19:12:34:178 send GP proxy commissioning mode
> 19:12:35:178 send permit join, duration: 0
> 19:12:35:178 send GP proxy commissioning mode
> 19:12:35:293 Search sensors done
> 19:12:36:177 send permit join, duration: 59
> 19:12:36:178 send GP proxy commissioning mode
> 19:12:37:178 send permit join, duration: 0
> 19:12:37:178 send GP proxy commissioning mode
> 19:12:37:939 Search sensors done
> 19:12:39:178 send permit join, duration: 59
> 19:12:39:178 send GP proxy commissioning mode
> 19:12:40:272 dev /dev/ttyS0 (/dev/ttyS0)
> 19:12:40:273 GW firmware version: 0x26520700
> 19:12:42:177 send permit join, duration: 0
> 19:12:42:178 send GP proxy commissioning mode
> 19:12:42:863 Search sensors done
> 19:12:44:177 send permit join, duration: 59
> 19:12:44:177 send GP proxy commissioning mode
> 19:12:44:242 Current channel 11
> 19:12:44:264 Device TTL 5532 s flags: 0x7
> 19:12:45:516 ZCL attribute report 0x680AE2FFFEE3C211 for cluster: 0x0006, ep: 0x01, frame control: 0x08, mfcode: 0x0000 
> 19:12:45:584 ZCL attribute report 0x680AE2FFFEE3C211 for cluster: 0x0008, ep: 0x01, frame control: 0x08, mfcode: 0x0000 
> 19:12:45:624 ZCL attribute report 0x680AE2FFFEE3C211 for cluster: 0x0300, ep: 0x01, frame control: 0x08, mfcode: 0x0000 
> 19:12:46:708 Search sensors done
> 19:12:47:177 send permit join, duration: 0
> 19:12:47:178 send GP proxy commissioning mode
> 19:12:48:178 send permit join, duration: 59
> 19:12:48:178 send GP proxy commissioning mode
> 19:12:49:178 send permit join, duration: 0
> 19:12:49:178 send GP proxy commissioning mode
> 19:12:49:403 Search sensors done
> 19:12:50:250 dev /dev/ttyS0 (/dev/ttyS0)
> 19:12:50:251 GW firmware version: 0x26520700
> 19:12:51:178 send permit join, duration: 59
> 19:12:51:178 send GP proxy commissioning mode
> 19:12:52:270 verified group capacity: 14 and group count: 2 of LightNode 0x00124b00210c17d1
> 19:12:52:270 0x00124b00210c17d1 found group 0xFFF0
> 19:12:52:270 0x00124b00210c17d1 found group 0x0001
> 19:12:52:311 Search sensors done
> 19:12:53:021 verified group capacity: 10 and group count: 3 of LightNode 0x680ae2fffee34f1e
> 19:12:53:021 0x680ae2fffee34f1e found group 0xFFF0
> 19:12:53:022 0x680ae2fffee34f1e found group 0x0002
> 19:12:53:022 0x680ae2fffee34f1e found group 0x0008
> 19:12:53:178 send permit join, duration: 0
> 19:12:53:178 send GP proxy commissioning mode
> 19:12:53:844 verified group capacity: 11 and group count: 2 of LightNode 0x680ae2fffee34c9b
> 19:12:53:844 0x680ae2fffee34c9b found group 0xFFF0
> 19:12:53:844 0x680ae2fffee34c9b found group 0x0003
> 19:12:54:178 send permit join, duration: 59
> 19:12:54:178 send GP proxy commissioning mode
> 19:12:54:569 verified group capacity: 11 and group count: 2 of LightNode 0x680ae2fffee3ca83
> 19:12:54:569 0x680ae2fffee3ca83 found group 0xFFF0
> 19:12:54:569 0x680ae2fffee3ca83 found group 0x0003
> 19:12:55:178 send permit join, duration: 0
> 19:12:55:178 send GP proxy commissioning mode
> 19:12:55:208 Search sensors done
> 19:12:55:392 verified group capacity: 11 and group count: 2 of LightNode 0x680ae2fffee3c211
> 19:12:55:392 0x680ae2fffee3c211 found group 0xFFF0
> 19:12:55:392 0x680ae2fffee3c211 found group 0x0003
> 19:12:56:000 verified group capacity: 23 and group count: 2 of LightNode 0x00178801025c89d8
> 19:12:56:000 0x00178801025c89d8 found group 0xFFF0
> 19:12:56:001 0x00178801025c89d8 found group 0x0002
> 19:12:56:805 verified group capacity: 8 and group count: 2 of LightNode 0x00158d0003101200
> 19:12:56:805 0x00158d0003101200 found group 0xFFF0
> 19:12:56:805 0x00158d0003101200 found group 0x0001
> 19:12:57:200 send permit join, duration: 0
> 19:12:57:200 send GP proxy commissioning mode
> 19:12:57:401 Search sensors done
> 19:12:57:544 verified group capacity: 23 and group count: 2 of LightNode 0x00178801025c8dee
> 19:12:57:545 0x00178801025c8dee found group 0xFFF0
> 19:12:57:545 0x00178801025c8dee found group 0x0005
> 19:12:58:940 [INFO] - No button handler for: TH01 endpoint: 0x01 cluster: 0x0402 command: 0x0A payload[0]: 000
> 19:12:58:940 ZCL attribute report 0x00124B002228C125 for cluster: 0x0402, ep: 0x01, frame control: 0x18, mfcode: 0x0000 
> 19:13:00:177 send permit join, duration: 59
> 19:13:00:177 send GP proxy commissioning mode
> 19:13:00:242 dev /dev/ttyS0 (/dev/ttyS0)
> 19:13:00:243 GW firmware version: 0x26520700
> 19:13:03:178 send permit join, duration: 0
> 19:13:03:178 send GP proxy commissioning mode
> 19:13:03:617 Search sensors done
> 19:13:04:177 remove outdated neighbor 0x0000
> 19:13:04:178 remove outdated neighbor 0x220E
> 19:13:04:178 remove outdated neighbor 0x1A98
> 19:13:04:178 remove outdated neighbor 0xF037
> 19:13:04:178 remove outdated neighbor 0xEC09
> 19:13:04:178 remove outdated neighbor 0xCD83
> 19:13:04:178 remove outdated neighbor 0xBAD0
> 19:13:04:178 remove outdated neighbor 0x8678
> 19:13:04:178 remove outdated neighbor 0x7020
> 19:13:05:178 send permit join, duration: 59
> 19:13:05:178 send GP proxy commissioning mode
> 19:13:07:179 send permit join, duration: 0
> 19:13:07:179 send GP proxy commissioning mode
> 19:13:07:756 Search sensors done
> 19:13:09:177 send permit join, duration: 59
> 19:13:09:177 send GP proxy commissioning mode
> 19:13:10:279 dev /dev/ttyS0 (/dev/ttyS0)
> 19:13:10:282 GW firmware version: 0x26520700
> 19:13:12:824 Search sensors done
> 19:13:13:177 send permit join, duration: 0
> 19:13:13:178 send GP proxy commissioning mode
> 19:13:14:178 send permit join, duration: 59
> 19:13:14:178 send GP proxy commissioning mode
> 19:13:17:177 send permit join, duration: 0
> 19:13:17:177 send GP proxy commissioning mode
> 19:13:17:888 Search sensors done
> 19:13:18:179 binding for cluster 0x0006 of 0x680AE2FFFEE3CA83 exists (verified by reporting)
> 19:13:18:179 skip configure report for cluster: 0x0006 attr: 0x0000 of node 0x680AE2FFFEE3CA83 (seems to be active)
> 19:13:18:179 binding for cluster 0x0008 of 0x680AE2FFFEE3CA83 exists (verified by reporting)
> 19:13:18:180 skip configure report for cluster: 0x0008 attr: 0x0000 of node 0x680AE2FFFEE3CA83 (seems to be active)
> 19:13:18:180 binding for cluster 0x0300 of 0x680AE2FFFEE3CA83 exists (verified by reporting)
> 19:13:18:180 skip configure report for cluster: 0x0300 attr: 0x0007 of node 0x680AE2FFFEE3CA83 (seems to be active)
> 19:13:18:180 skip configure report for cluster: 0x0300 attr: 0x0003 of node 0x680AE2FFFEE3CA83 (seems to be active)
> 19:13:18:180 skip configure report for cluster: 0x0300 attr: 0x0004 of node 0x680AE2FFFEE3CA83 (seems to be active)
> 19:13:18:180 skip configure report for cluster: 0x0300 attr: 0x0008 of node 0x680AE2FFFEE3CA83 (seems to be active)
> 19:13:20:181 send permit join, duration: 59
> 19:13:20:181 send GP proxy commissioning mode
> 19:13:20:291 dev /dev/ttyS0 (/dev/ttyS0)
> 19:13:20:294 GW firmware version: 0x26520700
> 19:13:22:295 Search sensors done
> 18:13:24:350 remove dead link for (8678, 276F)
> 18:13:24:706 send permit join, duration: 0
> 18:13:24:707 send GP proxy commissioning mode
> 18:13:24:830 remove dead link for (8678, 0)
> 18:13:25:006 remove dead link for (8678, BAD0)
> 18:13:25:311 remove dead link for (8678, EC09)
> 18:13:25:706 send permit join, duration: 59
> 18:13:25:707 send GP proxy commissioning mode
> 18:13:25:790 remove dead link for (8678, CD83)
> 18:13:26:082 remove dead link for (8678, F037)
> 18:13:26:270 remove dead link for (8678, 7020)
> 18:13:26:751 remove dead link for (8678, 220E)
> 18:13:27:157 remove dead link for (8678, 4532)
> 18:13:27:230 remove dead link for (8678, A0E)
> 18:13:27:710 remove dead link for (0, 276F)
> 18:13:28:190 remove dead link for (0, EC09)
> 18:13:28:231 remove dead link for (0, F037)
> 18:13:28:671 remove dead link for (0, BAD0)
> 18:13:28:708 send permit join, duration: 0
> 18:13:28:708 send GP proxy commissioning mode
> 18:13:28:873 Search sensors done
> 18:13:29:150 remove dead link for (0, CD83)
> 18:13:29:630 remove dead link for (0, 7020)
> 18:13:30:382 remove dead link for (276F, 7020)
> 18:13:30:591 remove dead link for (276F, BAD0)
> 18:13:31:070 remove dead link for (276F, CD83)
> 18:13:31:456 remove dead link for (276F, EC09)
> 18:13:31:550 remove dead link for (276F, F037)
> 18:13:31:706 send permit join, duration: 59
> 18:13:31:706 send GP proxy commissioning mode
> 18:13:31:790 dev /dev/ttyS0 (/dev/ttyS0)
> 18:13:31:791 GW firmware version: 0x26520700
> 18:13:32:110 remove dead link for (276F, 1A98)
> 18:13:32:531 remove dead link for (276F, C35F)
> 18:13:34:707 remove dead link for (CD83, 7020)
> 18:13:34:990 remove dead link for (CD83, BAD0)
> 18:13:35:471 remove dead link for (CD83, EC09)
> 18:13:35:707 send permit join, duration: 0
> 18:13:35:707 send GP proxy commissioning mode
> 18:13:35:781 remove dead link for (CD83, F037)
> 18:13:35:950 remove dead link for (EC09, F037)
> 18:13:36:143 Search sensors done
> 18:13:36:430 remove dead link for (EC09, BAD0)
> 18:13:36:856 remove dead link for (EC09, 7020)
> 18:13:36:911 remove dead link for (BAD0, 5D39)
> 18:13:37:390 remove dead link for (BAD0, 2DF8)
> 18:13:37:706 send permit join, duration: 59
> 18:13:37:706 send GP proxy commissioning mode
> 18:13:37:870 remove dead link for (BAD0, F037)
> 18:13:38:707 send permit join, duration: 0
> 18:13:38:707 send GP proxy commissioning mode
> 18:13:39:207 Search sensors done
> 18:13:39:551 reuse dead link (dead link container size now 32)
> 18:13:40:707 send permit join, duration: 59
> 18:13:40:707 send GP proxy commissioning mode
> 18:13:41:808 dev /dev/ttyS0 (/dev/ttyS0)
> 18:13:41:809 GW firmware version: 0x26520700
> 18:13:42:526 reuse dead link (dead link container size now 31)
> 18:13:43:486 reuse dead link (dead link container size now 30)
> 18:13:45:241 Search sensors done
> 18:13:45:407 reuse dead link (dead link container size now 29)
> 18:13:45:707 send permit join, duration: 0
> 18:13:45:707 send GP proxy commissioning mode
> 18:13:45:752 Current channel 11
> 18:13:45:781 Device TTL 5472 s flags: 0x7
> 18:13:45:886 reuse dead link (dead link container size now 28)
> 18:13:46:706 send permit join, duration: 59
> 18:13:46:706 send GP proxy commissioning mode
> 18:13:48:286 reuse dead link (dead link container size now 27)
> 18:13:48:708 send permit join, duration: 0
> 18:13:48:708 send GP proxy commissioning mode
> 18:13:49:561 Search sensors done
> 18:13:51:166 reuse dead link (dead link container size now 26)
> 18:13:51:706 send permit join, duration: 59
> 18:13:51:706 send GP proxy commissioning mode
> 18:13:51:774 dev /dev/ttyS0 (/dev/ttyS0)
> 18:13:51:775 GW firmware version: 0x26520700
> 18:13:55:606 Search sensors done
> 18:13:55:707 send permit join, duration: 0
> 18:13:55:707 send GP proxy commissioning mode
> 18:13:56:926 reuse dead link (dead link container size now 25)
> 18:13:57:707 send permit join, duration: 59
> 18:13:57:708 send GP proxy commissioning mode
> 18:13:59:659 Search sensors done
> 18:13:59:706 send permit join, duration: 0
> 18:13:59:708 send GP proxy commissioning mode
> 18:14:00:286 reuse dead link (dead link container size now 24)
> 18:14:01:708 send permit join, duration: 59
> 18:14:01:708 send GP proxy commissioning mode
> 18:14:01:818 dev /dev/ttyS0 (/dev/ttyS0)
> 18:14:01:819 GW firmware version: 0x26520700
> 18:14:02:783 reuse dead link (dead link container size now 23)
> 18:14:05:662 reuse dead link (dead link container size now 22)
> 18:14:05:706 send permit join, duration: 0
> 18:14:05:706 send GP proxy commissioning mode
> 18:14:06:015 Search sensors done
> 18:14:06:142 reuse dead link (dead link container size now 21)
> 18:14:06:793 [INFO] - No button handler for: TH01 endpoint: 0x01 cluster: 0x0001 command: 0x0A payload[0]: 021
> 18:14:06:793 ZCL attribute report 0x00124B002228C125 for cluster: 0x0001, ep: 0x01, frame control: 0x18, mfcode: 0x0000 
> 18:14:07:376 ZCL attribute report 0x680AE2FFFEE34F1E for cluster: 0x0300, ep: 0x01, frame control: 0x08, mfcode: 0x0000 
> 18:14:07:706 send permit join, duration: 59
> 18:14:07:706 send GP proxy commissioning mode
> 18:14:08:542 reuse dead link (dead link container size now 20)
> 18:14:08:707 send permit join, duration: 0
> 18:14:08:707 send GP proxy commissioning mode
> 18:14:09:023 reuse dead link (dead link container size now 19)
> 18:14:09:476 Search sensors done
> 18:14:11:422 reuse dead link (dead link container size now 18)
> 18:14:11:707 send permit join, duration: 59
> 18:14:11:707 send GP proxy commissioning mode
> 18:14:11:791 dev /dev/ttyS0 (/dev/ttyS0)
> 18:14:11:792 GW firmware version: 0x26520700
> 18:14:13:595 [INFO] - No button handler for: RWL021 endpoint: 0x02 cluster: 0x0001 command: 0x0A payload[0]: 021
> 18:14:13:596 ZCL attribute report 0x00178801021A4950 for cluster: 0x0001, ep: 0x02, frame control: 0x08, mfcode: 0x0000 
> 18:14:15:707 send permit join, duration: 0
> 18:14:15:707 send GP proxy commissioning mode
> 18:14:16:478 Search sensors done
> 18:14:17:662 reuse dead link (dead link container size now 17)
> 18:14:17:710 send permit join, duration: 59
> 18:14:17:710 send GP proxy commissioning mode
> 18:14:18:142 reuse dead link (dead link container size now 16)
> 18:14:18:706 send permit join, duration: 0
> 18:14:18:706 send GP proxy commissioning mode
> 18:14:19:561 Search sensors done
> 18:14:20:706 send permit join, duration: 59
> 18:14:20:707 send GP proxy commissioning mode
> 18:14:21:022 reuse dead link (dead link container size now 15)
> 18:14:21:800 dev /dev/ttyS0 (/dev/ttyS0)
> 18:14:21:803 GW firmware version: 0x26520700
> 18:14:23:903 reuse dead link (dead link container size now 14)
> 18:14:25:706 send permit join, duration: 0
> 18:14:25:706 send GP proxy commissioning mode
> 18:14:26:302 reuse dead link (dead link container size now 13)
> 18:14:26:604 Search sensors done
> 18:14:26:782 reuse dead link (dead link container size now 12)
> 18:14:28:708 send permit join, duration: 59
> 18:14:28:708 send GP proxy commissioning mode
> 18:14:29:182 reuse dead link (dead link container size now 11)
> 18:14:29:663 reuse dead link (dead link container size now 10)
> 18:14:29:707 send permit join, duration: 0
> 18:14:29:707 send GP proxy commissioning mode
> 18:14:29:791 Search sensors done
> 18:14:31:706 send permit join, duration: 59
> 18:14:31:706 send GP proxy commissioning mode
> 18:14:31:767 dev /dev/ttyS0 (/dev/ttyS0)
> 18:14:31:768 GW firmware version: 0x26520700
> 18:14:33:022 reuse dead link (dead link container size now 9)
> 18:14:33:502 reuse dead link (dead link container size now 8)
> 18:14:35:902 reuse dead link (dead link container size now 7)
> 18:14:36:707 send permit join, duration: 0
> 18:14:36:707 send GP proxy commissioning mode
> 18:14:36:950 Search sensors done
> 18:14:38:706 send permit join, duration: 59
> 18:14:38:707 send GP proxy commissioning mode
> 18:14:39:706 send permit join, duration: 0
> 18:14:39:707 send GP proxy commissioning mode
> 18:14:40:299 Search sensors done
> 18:14:41:663 reuse dead link (dead link container size now 6)
> 18:14:41:708 send permit join, duration: 59
> 18:14:41:708 send GP proxy commissioning mode
> 18:14:41:774 dev /dev/ttyS0 (/dev/ttyS0)
> 18:14:41:775 GW firmware version: 0x26520700
> 18:14:43:706 send permit join, duration: 0
> 18:14:43:707 send GP proxy commissioning mode
> 18:14:44:403 Search sensors done
> 18:14:44:893 [INFO] - No button handler for: RWL021 endpoint: 0x02 cluster: 0x0001 command: 0x0A payload[0]: 021
> 18:14:44:893 ZCL attribute report 0x0017880104F44026 for cluster: 0x0001, ep: 0x02, frame control: 0x08, mfcode: 0x0000 
> 18:14:45:759 Current channel 11
> 18:14:45:782 Device TTL 5412 s flags: 0x7
> 18:14:46:714 send permit join, duration: 59
> 18:14:46:714 send GP proxy commissioning mode
> 18:14:47:698 Search sensors done
> 18:14:47:706 send permit join, duration: 0
> 18:14:47:707 send GP proxy commissioning mode
> 18:14:49:707 send permit join, duration: 59
> 18:14:49:707 send GP proxy commissioning mode
> 18:14:50:706 send permit join, duration: 0
> 18:14:50:706 send GP proxy commissioning mode
> 18:14:50:718 Search sensors done
> 18:14:51:761 dev /dev/ttyS0 (/dev/ttyS0)
> 18:14:51:781 reuse dead link (dead link container size now 5)
> 18:14:51:782 GW firmware version: 0x26520700
> 18:14:52:706 send permit join, duration: 59
> 18:14:52:707 send GP proxy commissioning mode
> 18:14:54:706 send permit join, duration: 0
> 18:14:54:706 send GP proxy commissioning mode
> 18:14:54:782 Search sensors done
> 18:14:56:542 reuse dead link (dead link container size now 4)
> 18:14:57:023 reuse dead link (dead link container size now 3)
> 18:14:57:502 reuse dead link (dead link container size now 2)
> 18:14:58:942 reuse dead link (dead link container size now 1)
> 18:15:01:812 dev /dev/ttyS0 (/dev/ttyS0)
> 18:15:01:814 GW firmware version: 0x26520700
> 18:15:08:286 [INFO] - No button handler for: TH01 endpoint: 0x01 cluster: 0x0405 command: 0x0A payload[0]: 000
> 18:15:08:286 ZCL attribute report 0x00124B002228D384 for cluster: 0x0405, ep: 0x01, frame control: 0x18, mfcode: 0x0000 
> 18:15:11:798 dev /dev/ttyS0 (/dev/ttyS0)
> 18:15:11:800 GW firmware version: 0x26520700
> 18:15:21:774 dev /dev/ttyS0 (/dev/ttyS0)
> 18:15:21:775 GW firmware version: 0x26520700
> 18:15:25:823 reuse dead link (dead link container size now 0)
> 18:15:31:762 dev /dev/ttyS0 (/dev/ttyS0)
> 18:15:31:763 GW firmware version: 0x26520700
> 18:15:37:477 ZCL attribute report 0x680AE2FFFEE34F1E for cluster: 0x0006, ep: 0x01, frame control: 0x08, mfcode: 0x0000 
> 18:15:38:481 ZCL attribute report 0x680AE2FFFEE34F1E for cluster: 0x0008, ep: 0x01, frame control: 0x08, mfcode: 0x0000 
> 18:15:41:832 dev /dev/ttyS0 (/dev/ttyS0)
> 18:15:41:835 GW firmware version: 0x26520700
> 18:15:45:748 Current channel 11
> 18:15:45:770 Device TTL 5352 s flags: 0x7
> 18:15:51:795 dev /dev/ttyS0 (/dev/ttyS0)
> 18:15:51:796 GW firmware version: 0x26520700
> 18:15:57:024 Websocket disconnected 10.13.37.22:60315, state: 0, close-code: 1001, reason: 
> 18:15:58:501 New websocket 10.13.37.22:60354 (state: 3) 
> 18:16:01:785 dev /dev/ttyS0 (/dev/ttyS0)
> 18:16:01:785 GW firmware version: 0x26520700
> 18:16:11:813 dev /dev/ttyS0 (/dev/ttyS0)
> 18:16:11:814 GW firmware version: 0x26520700
> 18:16:21:768 dev /dev/ttyS0 (/dev/ttyS0)
> 18:16:21:770 GW firmware version: 0x26520700
> 18:16:30:947 [INFO] - No button handler for: DS01 endpoint: 0x01 cluster: 0x0001 command: 0x0A payload[0]: 021
> 18:16:30:947 ZCL attribute report 0x00124B001F90F55A for cluster: 0x0001, ep: 0x01, frame control: 0x18, mfcode: 0x0000 
> 18:16:31:116 [INFO] - No button handler for: TH01 endpoint: 0x01 cluster: 0x0405 command: 0x0A payload[0]: 000
> 18:16:31:116 ZCL attribute report 0x00124B002228C125 for cluster: 0x0405, ep: 0x01, frame control: 0x18, mfcode: 0x0000 
> 18:16:31:829 dev /dev/ttyS0 (/dev/ttyS0)
> 18:16:31:832 GW firmware version: 0x26520700
> 18:16:41:806 dev /dev/ttyS0 (/dev/ttyS0)
> 18:16:41:808 GW firmware version: 0x26520700
> 18:16:45:745 Current channel 11
> 18:16:45:768 Device TTL 5292 s flags: 0x7
> 18:16:51:156 ZCL attribute report 0x680AE2FFFEE3CA83 for cluster: 0x0006, ep: 0x01, frame control: 0x08, mfcode: 0x0000 
> 18:16:51:239 ZCL attribute report 0x680AE2FFFEE3CA83 for cluster: 0x0300, ep: 0x01, frame control: 0x08, mfcode: 0x0000 
> 18:16:51:293 ZCL attribute report 0x680AE2FFFEE3CA83 for cluster: 0x0008, ep: 0x01, frame control: 0x08, mfcode: 0x0000 
> 18:16:51:776 dev /dev/ttyS0 (/dev/ttyS0)
> 18:16:51:777 GW firmware version: 0x26520700
> ^A18:17:00:878 ZCL attribute report 0x680AE2FFFEE34F1E for cluster: 0x0300, ep: 0x01, frame control: 0x08, mfcode: 0x0000 
> 18:17:01:790 dev /dev/ttyS0 (/dev/ttyS0)
> 18:17:01:794 GW firmware version: 0x26520700
> 18:17:10:556 ZCL attribute report 0x680AE2FFFEE34F1E for cluster: 0x0300, ep: 0x01, frame control: 0x08, mfcode: 0x0000 
> 18:17:11:778 dev /dev/ttyS0 (/dev/ttyS0)
> 18:17:11:780 GW firmware version: 0x26520700
> 18:17:21:787 dev /dev/ttyS0 (/dev/ttyS0)
> 18:17:21:790 GW firmware version: 0x26520700
> 18:17:23:029 ZCL attribute report 0x680AE2FFFEE34C9B for cluster: 0x0006, ep: 0x01, frame control: 0x08, mfcode: 0x0000 
> 18:17:23:081 ZCL attribute report 0x680AE2FFFEE34C9B for cluster: 0x0008, ep: 0x01, frame control: 0x08, mfcode: 0x0000 
> 18:17:23:120 ZCL attribute report 0x680AE2FFFEE34C9B for cluster: 0x0300, ep: 0x01, frame control: 0x08, mfcode: 0x0000 
> 18:17:28:078 [INFO] - No button handler for: TH01 endpoint: 0x01 cluster: 0x0402 command: 0x0A payload[0]: 000
> 18:17:28:078 ZCL attribute report 0x00124B002228D384 for cluster: 0x0402, ep: 0x01, frame control: 0x18, mfcode: 0x0000

## Additional context
Zigbee network has a mixture of various brands of bulb, and a mix of xaiomi mijia and sonoff sensors, switches

**Comment by GeoCyberwolf** [comment]
Update: after opening the network in deconz gui, trying to add the switch(which didnt work) then closing it. I attempted again to add the switch in the webapp, which then worked. So something about opening and closing the network in the gui allowed the switch to be added. 

I am still unable to add it to a group to control lights though. I am happy to send one of these switches to a developer for testing if it helps get these switches working. I have 4 of them and they are currently useless.

**Comment by accelle17** [comment]
@GeoCyberwolf were you able to add the snzb-01 to a group after #3502 or 2.05.87?

**Comment by GeoCyberwolf** [comment]
@accelle17 No, unfortunately it doesn't show on the list when I go to add a switch to a group. I tried removing and re-adding the switch since updating to .88

**Comment by SwoopX** [comment]
Please check the old webapp and have a look if it shows up there.

**Comment by accelle17** [comment]
> Please check the old webapp and have a look if it shows up there.

Great. can see it on the old web app. Should this be the behavior?

**Comment by GeoCyberwolf** [comment]
@SwoopX How do I access the old webapp? I have just been browsing to the hostname of the raspberry pi deconz is running on.

**Comment by SwoopX** [comment]
Should be http://YOURIP/login.html

@accelle17 yes. What you see there are the zigbee groups. By using them, the coordinator must not be online. Never understood why they are not properly exposed via Phoson though.

**Comment by GeoCyberwolf** [comment]
@SwoopX Ah, that works, thanks. I can add lights to the switch group in the old webapp, and the switch will control those lights. 

Is there any way to get the switch working in the new webapp, like my hue dimmer switches already do?

**Comment by sunshineautomate** [comment]
Guys, This bug is still an issue. Is there any change I it can get some attention please.

**Comment by GeoCyberwolf** [comment]
> 
> 
> Guys, This bug is still an issue. Is there any chance I it can get some attention please.

You can do it in the old deconz webui, just not the phoscon webapp (Oh my god this is such a fragmented piece of software!!). Try going to http://[ip address]/login.html. Should give you the old WebUI where you can link the switch.

Though I would suggest opening another issue to get support properly added to the phoscon webapp.
